CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## [Unreleased]
 
+## [4.0.1] - 2018-06-23
+
+### Changed
+
+-   Use `var` instead of `let` + `'use strict'` ([**@ralphtheninja**](https://github.com/ralphtheninja))
```

Comment:
This reads as "use var instead of use strict", but no biggie

Reply:
I'll split into two.